### ISSUE TITLE
Caregiver revisions based on program & Veteran feedback

### ISSUE THREAD

**Issue by jonathanjnelson** [department-of-veterans-affairs/va.gov-team#12806]
## Issue Description
During beta testing with online 10-10CG, Caregiver Support Coordinators and Veterans/caregivers had feedback regarding certain screens. We have made design changes that the program has approved, and want to run them by VSP Content/Design/A11y leads before we code them.

We have included the new designs below. Program feedback that we integrated includes:

- Intro page
     - Changed phone number from VA.gov contact center (877-222-8387) to HEC help line (855-488-8440) in the Step 1: Prepare section and in the footer
     - Also removed references to VSO support in the Step 1: Prepare section
     - Under Step 2: Apply - We added an extra line to explain that some questions are "required"
     - Under Step 3: Next steps - We added three extra sentences about the appeal process. 

- Gender question (for all Veterans and caregivers)
     - Restricted the options to only male or female (removed "unknown") but made the question not required

- Medical center question (for Veteran)
     - Veterans were confused by the two questions, which seemed very similar. We added an additional heading, changed the wording of both headings, and bolded some text to distinguish between the two. Both questions are mandated to appear on the online form. 

- Open question
     - The program team asked specific questions around the capitalization of "Family Caregiver" in a couple of spots on the intro page. What guidance can we provide them with or should the capitalization be changed? For context, "Family Caregiver" is capitalized as such throughout their documentation.

# Screenshots 
## Intro 

![Intro-Annoated](https://user-images.githubusercontent.com/58444931/91614189-14455e00-e94f-11ea-893c-d09ed95e6506.png)

## Gender

![Vet](https://user-images.githubusercontent.com/58444931/91614279-3c34c180-e94f-11ea-81a4-bef6fc6381f2.png)

## Medical center

![Vet Medical Center-care](https://user-images.githubusercontent.com/58444931/91886302-c3837d00-ec4e-11ea-966f-fecf708d3112.png)

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by ala-yna** [comment]
@psanders86 - Here is the new ticket! Let me know if you have any questions/comments/feedback/concerns, otherwise we'll pass it along to the other VSP folks. Thank you!

**Comment by psanders86** [comment]
# Medical center #

VA.gov has a guide on [how we should use bold text](https://design.va.gov/content-style-guide/bold-text). We don't perfectly align with that guidance, but I think we come close if we make the second heading more specific and expand the bold texts so they are easily identified as design elements that correspond with headings.

![image](https://user-images.githubusercontent.com/59177093/91752982-29abc980-eb95-11ea-9bac-f93a516cc62b.png)


### Recent medical care ###

Please enter the name of the medical facility where the Veteran **last received medical treatment.**

### Future VA services ###

Please select the VA medical center or clinic where the Veteran **receives or plans to receive health care services.**




# Capitalization of family caregiver #

There was extensive conversation this spring with VSP content folks about the program's desire to capitalize "family caregiver" as it is on the paper form. We came to the conclusion that it would be OK to capitalize specific titles, such as Primary Family Caregiver, Secondary Family Caregiver, and Caregiver Support Coordinator as they are unique to the Caregiver program. But we would be unable to capitalize the generic phrase.

**Rationale:** We rely on [VA.gov's content style guide](https://design.va.gov/content-style-guide/capitalization), which was created to provide a consistent user experience across VA.gov's websites and also to ensure readability, particularly among Veterans who may have cognitive challenges. Content with excessive capitalization is more difficult to understand for people with cognitive issues, which is why VA.gov uses sentence case capitalization.

Notably for this topic, the style guide specifically references "family caregiver" as [a shorthand term](https://design.va.gov/content-style-guide/word-list) that it is not capitalized.

![image](https://user-images.githubusercontent.com/59177093/91749961-4abdeb80-eb90-11ea-8cc4-2bc0aa5ea59b.png)

The family caregiver term is also being used without capitalization on the [recently updated PCAFC landing page](https://www.va.gov/family-member-benefits/comprehensive-assistance-for-family-caregivers/) that was updated by the VA.gov team in coordination with the program team.

![image](https://user-images.githubusercontent.com/59177093/91750501-33cbc900-eb91-11ea-91aa-0f3de020c161.png)

**Next steps:** There is already an ongoing style guide discussion within VA.gov. If the program team would like to advocate changing VA.gov style so that the capitalization of "family caregiver" is added to the style guide, we can connect them with the appropriate team.

**Comment by jonathanjnelson** [comment]
@ala-yna I updated the page to reflect Patrick's recommendations. 

![Vet Medical Center-care](https://user-images.githubusercontent.com/58444931/91886302-c3837d00-ec4e-11ea-966f-fecf708d3112.png)

**Comment by ala-yna** [comment]
Good morning! Since the Caregiver team already had our staging review, we're reaching out to get 👀 on a couple of small content changes that we're making on behalf of direct feedback and requests from our business owners. Those requested changes are listed in the issue description, and reflected (as prototype screenshots) in the screenshots provided in the issue description. I would love to get feedback by EOD today or tomorrow (9/3) so that we can push these changes through to our staging and production environments, where we are currently engaged in beta testing. Thank you!

Tagging: @peggygannon, @DanielleThierryUSDSVA, @emilywaggoner, @CrystabelReiter, @sshein, @1Copenut (since @jenstrickland is out) for 👀 , and @andreahewitt-odd, @ccgongie, @mlighdc, and @LinnJS for awareness.

(A note: the screenshots have been updated to reflect @psanders86's comments above)

**Comment by jenstrickland** [comment]
Waiting for a flight, I had a few minutes while waiting on a meeting change confirmation, so I took a quick look. cc @1Copenut @ala-yna 

**Consider** adding the option to "save" as well as print:
> After submitting the form, you'll receive a confirmation screen that you can print **or save** for your records.

**Must** add a label to the medical center input:

<img width="440" alt="Screen Shot 2020-09-02 at 12 09 43 PM" src="https://user-images.githubusercontent.com/57469/92008744-89c97980-ed15-11ea-8460-9e5b2b420deb.png">

**Comment by DanielleThierryUSDSVA** [comment]
Hi @ala-yna.

@psanders86 gave a good explanation of the capitalization question around "family caregiver" above. This issue was settled with the program team as a compromise so I would recommend following the agreed-upon approach of:

- Initial cap "Primary Family Caregiver" and "Secondary Family Caregiver" when used to refer to those specific roles in terms of the program
- Lowercase "family caregiver" when speaking more broadly (in alignment with AP style and with the VA.gov content style guide)

I'll sync with @peggygannon on the other questions here so we can give consolidated feedback. Thanks!

**Comment by 1Copenut** [comment]
@ala-yna I took a look and starred Jennifer's comment. No additional accessibility feedback.

**Comment by peggygannon** [comment]
Hi, @ala-yna Danielle and I synced this morning, and I have just some minor recommended content tweaks to tighten up language: 

Under **Next steps**, I recommend updating this sentence TO: 
"You can also contact your Caregiver Support Coordinator if you have questions." 

Under **Need help**, I recommend updating this sentence TO: 
"If you have questions about completing your application, please call [855-488-8440] or contact your local Caregiver Support Coordinator.

Under **Recent medical care**, I recommend updating the label "Was this a hospital or clinic?" TO "Is this a hospital or clinic?"

**Comment by emilywaggoner** [comment]
Thanks @ala-yna and @jonathanjnelson - the outline of the changes and highlighted screenshots were super clear and easy to follow. Also, I support Jennifer's recommendation to add the label to the Medical Center field.

The only feedback I have is a **Consider** item:
On the page with the Medical Center questions, consider tightening up the space between the headings and subsequent body text to emphasize the relationship between the two pieces of content. Screenshot below to illustrate, but let me know if you have questions!

![image](https://user-images.githubusercontent.com/1379629/92259563-4c93f180-eea5-11ea-8204-f1b31ac11a3c.png)

That's all I got, thanks again!

**Comment by jonathanjnelson** [comment]
Hi @LinnJS!
![Vet Medical Center-care](https://user-images.githubusercontent.com/58444931/92621192-21384a80-f289-11ea-928d-534c91670b31.png)

**Comment by LinnJS** [comment]
Attaching this [InVision](https://vsateams.invisionapp.com/share/U4YJL36QA35#/screens/429945712) for reference

**Comment by jonathanjnelson** [comment]
Updated designs can be seen in [InVision](https://vsateams.invisionapp.com/share/XBYROF0SC3U#/431602920_Intro_)
